specs/agents/mobile/README.md

```diff
@@ -20,56 +20,57 @@ However, the Elastic mobile agents don't set every attribute defined (many only
+| [`telemetry.sdk.language`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/#telemetry-sdk)| `service.language.name`   | `swift`, `java`              | :white_check_mark: yes | Elastic's `agent.name` is mapped from:  `telemetry.sdk.name`/`telemetry.sdk.language` |
+| [`os.description`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)       | `host.os.full`             | `iOS Version 15.5 (Build 19F70)`      | :x: no | |
+| [`os.type`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)             | `host.os.platform` | `darwin` | :white_check_mark: yes |  |
+| [`os.version`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)           | `host.os.version` | `15.5.0`        | :x: no | |
+| [`os.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)              | `host.os.name` and `host.os.type` | `iOS`, `Android`   | :white_check_mark: yes |  |
+| [`deployment.environment`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/deployment_environment/) | `service.environment`    | `production`, `dev`              | :x: no |    |
+| [`deivce.id`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)        | `device.id`        | `E733F41E-DF47-4BB4-AAF0-FD784FD95653` | :x: no |  [Follow this description.](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/) (Device ID should be fix and unique for a device but should not carry PII)  |
+| [`device.model.identifier`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/) | `device.model.identifier` | `iPhone4`,`SM-G920F`            | :x: no  |             |
+| [`device.model.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)       | `device.model.name` | `Samsung Galaxy S6`            | :x: no  | This information is potentially not directly available on the device and needs to be derived / mapped from `device.model.identifier`. In this case, the APM server should do the mapping. |
+| [`device.manufacturer`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)     | `device.manufacturer` | `Apple`, `Samsung`            | :x: no  |  This information is potentially not directly available on the device and needs to be derived / mapped from `device.model.identifier`. In this case, the APM server should do the mapping. |   
+| [`process.runtime.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/process/#process-runtimes) | `service.runtime.name` | TBD: `???` | :x: no | The name of the runtime |
```

Comment:
@bryce-b , @LikeTheSalad 
What do you think would be meaningful values for `process.runtime.name` and `process.runtime.version` for iOS and Android respectively?

Would there be additional information or would we use the same as in `os.*` fields? In the latter case we then could omit writing `process.*` fields and then just let the APM server map the `os.*` fields also to `service.runtime.*` fields in addition in the case of mobile agents.

Reply:
If I understood correctly, for Android it'd be the custom [Android virtual machine](https://source.android.com/docs/core/dalvik), which is always the same on an android device. I checked and it seems like we can get a version of it easily, although I believe it's always a 1-1 mapping with the OS version so I'm not sure if it's too useful. Usually when dealing with bugs related to Android versions, devs tend to check the Android OS version itself, but so far I haven't seen a case where it'd be needed to check the Dalvik/ART version too.